lib/fluent/log.rb

```diff
@@ -281,6 +284,7 @@ def event(level, args)
       }
 
       map.each_pair {|k,v|
+        next if (k.to_s == "plugin_type" || k.to_s == "plugin_id") and @optional_header
```

Comment:
I decided these lines leave as-is.
Because shutting down Fluentd is rare operation than other operations.
And outputting `plugin` and `plugin_id` information improves understandability.